### ISSUE TITLE
12GB GPU memory is not enough for training with default config

### ISSUE THREAD

**Issue by qiyancos** [qcraftai/simtrack#8]
I have a gpu card with only 12GB video memory. The process failed with out-of-memory error when I tried to run with default config. What should I change in the default config to run your model normally?

**Comment by xiaodongyang** [comment]
You may try with a smaller batch size and adjust the learning rate. AMP also helps.
